src/main/java/bio/terra/service/snapshotbuilder/SnapshotRequestDao.java

```diff
@@ -45,6 +46,7 @@ public class SnapshotRequestDao {
           new SnapshotAccessRequestResponse()
               .id(rs.getObject(ID, UUID.class))
               .datasetId(rs.getObject(DATASET_ID, UUID.class))
```

Comment:
Can you remove `datasetId` from this API and from the database at this point? We can continue to support the dataset version of the endpoint but not write the record to the database and not return it in the API response, as the UI doesn't do anything with the response right now.

Reply:
Do you mean remove all the dao methods that use database_id?

Reply:
And then replace their usages in the service methods with the new snapshot id dao methods?

Reply:
I think it's okay to leave the scope of this ticket narrow and only include the incremental step of adding the snapshot related operations.

Reply:
I was thinking you'd remove the DAO calls and remove the service code that calls the DAO, but not hook up the new API calls yet, if that was work to defer until later.

Reply:
Okay, I think I will stick to the original scope of the ticket and file a ticket for the next iteration of work that removes the datasetid usage